src/app/services/server-api.service.ts

```diff
@@ -30,28 +32,37 @@ export class ServerApiService {
   private createHeaders(queryParams?: Map<string, string>) {
     let parameters = new HttpParams();
     if (queryParams) {
-      queryParams.forEach((value, key , map) => {
+      queryParams.forEach((value, key, map) => {
         parameters = parameters.set(key, value);
       });
     }
     return {
       params: parameters,
       headers: new HttpHeaders({
         'Accept': 'application/json',
-        'X-USER-ID': this.getAuthorization(),
+        'X-USER-ID': this.getUserId(),
         'X-SESSION-ID': this.getSessionId()
       })
     };
   }
 
-  // TODO: Add appropriate authorization that should be collected when a Audiologist or Admin Logs in.
-  // TODO: We will pull this into sepperate https interceptor at a future point.
-  private getAuthorization(): string {
-    return '15';
+    private getUserId(): string {
+    let value = Utilities.getSessionStorage('userId');
+    if (value === null) {
+      return ' ';
+    }
+    return value;
+
   }
 
   private getSessionId(): string {
-    return '15';
+
+    let value = Utilities.getSessionStorage('sessionId');
+    if (value === null) {
+      return ' ';
+    }
+    return value;
   }
 
+  // TODO: Add hearbeat function
 }
```

Comment:
getUserId() and getSessionId() do exactly the same thing.  Maybe combine into a getId() and pass in 'userId' or 'sessionId'

Reply:
That's a REALLY good suggestion. I'll use that for the revision.